lighthouse-core/audits/third-party-summary.js

```diff
@@ -128,6 +133,15 @@ class ThirdPartySummary extends Audit {
     const summary = {wastedBytes: 0, wastedMs: 0};
 
     const results = Array.from(summaryByEntity.entries())
+      .filter(([entity]) => {
+        return !entity.domains.find(domain => {
+          if (domain.startsWith('*.')) {
```

Comment:
I don't think we need to be doing all that, just check if the entity is the same as the mainEntity, no? I can't remember if the entity itself will be referentially equal, so might need to check the name or something but hopefully this gives the gist :)

```js
const mainEntity = ThirdPartySummary.getEntitySafe(mainResource.url);
// ...

const results = Array.from(summaryByEntity.entries())
  .filter(([entity]) => entity !== mainEntity)

Reply:
🤦‍♂ I made that more complicated than it needed to be! Updated to compare entities.